public/app/plugins/panel/graph/tab_annotations.html

```diff
@@ -0,0 +1,16 @@
+<div class="editor-row">
+  <div class="gf-form-group">
+    <div class="gf-form-inline">
+      <gf-form-switch class="gf-form" label="Match any" label-class="width-9" checked="ctrl.panel.options.annotation.matchAny" on-change="ctrl.refresh()"
+        tooltip="By default Grafana only shows annotations that match all tags. Enabling this shows annotations that match any of the tags."></gf-form-switch>
+      <span class="gf-form-label">
+        Tags
+        <info-popover mode="right-normal">
+          Tag filter entered here can include regular expressions and template variables, e.g. "(foo|bar:$var|other:.*)"
+        </info-popover>
+      </span>
+      <bootstrap-tagsinput ng-model="ctrl.panel.options.annotation.tags" tagclass="label label-tag" placeholder="add tags">
+      </bootstrap-tagsinput>
+    </div>
+  </div>
+</div>
```

Comment:
Hmm, I can understand that we might be hesitant adding more html/Angular code as we're really doing a lot of React migrations right now.

Reply:
It would be hard to include React migration inside this PR.
Other tabs are also using html/Angular code and this new html/Angular file is adapted from [public/app/plugins/datasource/grafana/partials/annotations.editor.html](https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/grafana/partials/annotations.editor.html#L28-L50).

Reply:
I totally understand that @pbaumard and to be clear my comment was more of a "thinking out loud" for myself type of thing. I think this looks good, but not sure what @dprokop or @torkelo thinks?

Reply:
Sorry I meant @ryantxu .

Reply:
How close is this to getting completed? I am really looking forward to this feature :-D